### ISSUE TITLE
Localized Folders

### ISSUE THREAD

**Issue by tomglazer89** [Syroot/KnownFolders#3]
Hi RayKoopa,
Thanks for the app, its really a time saving creation.

i just wanted to let you know i found few problems after using your class to change my User Shell directories.

i'm sure the reason why you haven't noticed this problem is that you are using an **English** version of windows.

after changing certain directories on my localized Windows such as:

- Documents
- Videos
- Pictures
- Music
- Downloads
- ThisPCDesktop

i've notice their description  changed to English instead of the localized directory description, and that some "links" to these folders on my system were broken.
the reason for that is because Windows uses another shell (Localized shell) for other languages then English

so needs to add these:
(I'm not sure if this has  a different Guid for each language?)

          [KnownFolderGuid("F42EE2D3-909F-4907-8871-4C22FC0BF756")]
          LocalDocuments,

          [KnownFolderGuid("35286A68-3C57-41A1-BBB1-0EAE73D76C95")]
          LocalVideos,

          [KnownFolderGuid("0DDD015D-B06C-45D5-8C4C-F59713854639")]
          LocalPictures,

         [KnownFolderGuid("A0C69A99-21C8-4671-8703-7934162FCF1D")]
          LocalMusic,

          [KnownFolderGuid("7d83ee9b-2244-4e70-b1f5-5393042af1e4")]
          LocalDownloads,

          [KnownFolderGuid("754AC886-DF64-4CBA-86B5-F7FBF4FBCEF5")]
          ThisPCDesktopFolder,

then, if you want to change your shell redirection for Music folder for example, you need to change the "Music" shell AND the "LocalMusic" shell as well. same for others above..

also, in the latest Windows update another folder was added called "3D Objects". just letting you know..

        [KnownFolderGuid("31C0DD25-9439-4F12-BF41-7FF4EDA38722")]
        _3DObjects

Thanks again, Tom

**Comment by RayKoopa** [comment]
Ensure to call `Create` after changing the path, so it can set up the `desktop.ini` file storing the localized value etc.

**Comment by RayKoopa** [comment]
I looked into this and added several new IDs introduced with Windows 10. However, I couldn't find your `LocalXxx` IDs from the [official list](https://docs.microsoft.com/en-us/windows/desktop/shell/knownfolderid). Where are they described?

**Comment by tomglazer89** [comment]
well, they are not described there at all. i have no idea why. 
but there i s something about it on this page:
https://console.kim.sg/windows-common-shell-commands/

you'll never see them if you are using an English version of Widows. but I've noticed this value created in the registry when i am using windows to redirect these folders.
i just worry that maybe these guids are language specific to Hebrew, and could be different for other languages.

also attached my Folder Descriptions in registry, look for line 929 you'll see "Name"="Local Music" in there

shell:Local Music
shell:Local Downloads
shell:Local Pictures
shell:Local Videos
shell:Local Documents

these folders are the only to have "local" version.

[‏‏FolderDescriptions.txt](https://github.com/Syroot/KnownFolders/files/2353758/FolderDescriptions.txt)

        /// <summary> -> Added by Tom
        /// The per-user Local Pictures folder.
        /// Defaults to &quot;%USERPROFILE%\Pictures&quot;.
        /// </summary>
        [KnownFolderGuid("0DDD015D-B06C-45D5-8C4C-F59713854639")]
        LocalPictures,

        /// <summary> -> Added by Tom
        /// The per-user Local Videos folder.
        /// Defaults to &quot;%USERPROFILE%\Videos&quot;.
        /// </summary>
        [KnownFolderGuid("35286A68-3C57-41A1-BBB1-0EAE73D76C95")]
        LocalVideos,

        /// <summary> -> Added by Tom
        /// The per-user Local Documents folder.
        /// Defaults to &quot;%USERPROFILE%\Documents&quot;.
        /// </summary>
        [KnownFolderGuid("F42EE2D3-909F-4907-8871-4C22FC0BF756")]
        LocalDocuments,
        /// <summary> -> Added by Tom
        /// The per-user Local Downloads folder.
        /// Defaults to &quot;%USERPROFILE%\Downloads&quot;.
        /// </summary>
        [KnownFolderGuid("7d83ee9b-2244-4e70-b1f5-5393042af1e4")]
        LocalDownloads,
        /// <summary> Added by Tom
        /// The per-user Music folder.
        /// Defaults to &quot;%USERPROFILE%\Music&quot;.
        /// </summary>
        [KnownFolderGuid("A0C69A99-21C8-4671-8703-7934162FCF1D")]
        LocalMusic,

**Comment by RayKoopa** [comment]
That's pretty weird. It _feels_ like MS doesn't want programs to change those paths, and keep that a task which has to be manually done via their horrible Win9x folder property page UI.

However, I searched the web for them, and I saw the issue also with other non-English, but also non-Hebrew systems, so I added those IDs and will push an update in a few. Also added the `Objects3D` and the many new UWP app folders.
